docs/pages/versions/unversioned/sdk/notifications.md

```diff
@@ -1618,6 +1619,23 @@ export interface WeeklyTriggerInput {
 }
 ```
 
+#### `YearlyTriggerInput`
```

Comment:
do we also need to add `YearlyNotificationTrigger` to the docs? (already added in the readme)

Reply:
sure! I missed it because Weekly didn't have an entry there either (but Daily has), so I'm adding both missing ones